apps/src/music/views/MusicPlayView.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {memo, useCallback, useMemo} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import {Heading2, BodyTwoText} from '@cdo/apps/componentLibrary/typography';
+import moduleStyles from './music-play-view.module.scss';
+import Link from '@cdo/apps/componentLibrary/link';
+
+import musicI18n from '../locale';
+
+const codeOrgLogo = require(`@cdo/static/music/code-dot-org-white-logo.png`);
```

Comment:
Should we put this in a shared location so that other parts of the site can use it, too?  Also, can we use an SVG?